tests/vmi_multus_test.go

```diff
@@ -602,6 +618,9 @@ var _ = Describe("[Serial]SRIOV", func() {
 	}
 
 	tests.BeforeAll(func() {
+		cloudInitNetworkData, err = libnet.CreateDefaultCloudInitNetworkData()
+		Expect(err).ToNot(HaveOccurred())
+
```

Comment:
Previously there was a function used only once that wrapped this. Since function was used only once, I just removed it and in place pasted content

Reply:
I think the confusing thing is that you do this in two tests, and in the rest you use this one.
So I guess it makes sense, but you need to remove the ones that appear in the tests as well.

I would also place it in a `BeforeEach` instead, to make sure no one mutates this data.  In general we now try to avoid using `BeforeAll` because it has no counter-part (`AfterAll`), resulting in flaky results.

Reply:
So, my understanding is that you want this to be moved to BeforeEach, right?

Reply:
That one was one, yes. The other was not to call it also in the other two tests.. Hope I did not missed some hidden reason there.